components/skus/browser/skus_sdk_service.h

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#ifndef BRAVE_COMPONENTS_SKUS_BROWSER_SKUS_SDK_SERVICE_H_
+#define BRAVE_COMPONENTS_SKUS_BROWSER_SKUS_SDK_SERVICE_H_
+
+#include <memory>
+#include <string>
+
+#include "base/memory/weak_ptr.h"
+#include "brave/components/skus/browser/brave-rewards-cxx/src/wrapper.h"
+#include "brave/components/skus/common/skus_sdk.mojom.h"
+#include "components/keyed_service/core/keyed_service.h"
+
+class PrefService;
+
+namespace network {
+class SharedURLLoaderFactory;
+}  // namespace network
+
+// This service is wrapping the calls to our SKU SDK
+//
+// For more information, please see:
+// https://github.com/brave-intl/br-rs/tree/skus
+class SkusSdkService : public KeyedService {
```

Comment:
similar to https://github.com/brave/brave-core/pull/10847/files#r742280749 this seems like it could grow out of control like Ads and Rewards services. Can we break this down into more meaningful parts?

Reply:
I guess we need something to own the rust sdk, but maybe that should be internal to the skus component? Also can we drop `Sdk` in general from these class names?

Reply:
Renaming done in https://github.com/brave/brave-core/pull/10847/commits/0c492c6a95c3b11eb0f4ab4c316510b78a030d19